cmd/util/ledger/migrations/atree_register_migration_test.go

```diff
@@ -21,11 +21,6 @@ func TestAtreeRegisterMigration(t *testing.T) {
 	log := zerolog.New(zerolog.NewTestWriter(t))
 	dir := t.TempDir()
 
-	validation := migrations.NewCadenceDataValidationMigrations(
```

Comment:
Is this not needed anymore?

Reply:
No, validation is done differently now, this was just a thing from the past we forgot to remove. The validation is now in cmd/util/ledger/migrations/cadence_value_validation.go.